wgpu-hal/src/gles/egl.rs

```diff
@@ -580,6 +580,25 @@ pub struct Instance {
     inner: Mutex<Inner>,
 }
 
+impl Instance {
+    pub fn raw_display(&self) -> MappedMutexGuard<'_, egl::Display> {
```

Comment:
Is it copyable?  Can we just return `egl::Display` here?

Reply:
Yes it is copy.